disnake/abc.py

```diff
@@ -1099,7 +1099,7 @@ async def create_invite(
             unique=unique,
             target_type=target_type.value if target_type else None,
             target_user_id=target_user.id if target_user else None,
-            target_application_id=target_application_id,
+            target_application_id=target_application.value if target_application else None,
```

Comment:
Actually, `try_enum_to_int` would be much better here.

Reply:
Okay, lemme fix it

Reply:
while you're at that do it for `target_type` too